File: oop_chess/src/chess/beans/ValidationResponse.java
package chess.beans;

public class ValidationResponse {
    private boolean valid;
    private String message;

    public ValidationResponse(boolean valid) {
        this.valid = valid;
    }

    public ValidationResponse(boolean valid, String message) {
        this.valid = valid;
        this.message = message;
    }

    public boolean isValid() {
        return valid;
    }

    public void setValid(boolean valid) {
        this.valid = valid;
    }

    public String getMessage() {
        return message;
    }

    public void setMessage(String message) {
        this.message = message;
    }
}

File: chess_cpp/beans/ValidationResponse.cpp
//
// Created by Hrach Khachatryan on 12/15/19.
//

#include "ValidationResponse.h"
ValidationResponse::ValidationResponse(bool valid) {
    this->valid = valid;
}

ValidationResponse::ValidationResponse(bool valid, std::string message) {
    this->valid = valid;
    this->message = message;
}

bool ValidationResponse::isValid() {
    return valid;
}

void ValidationResponse::setValid(bool valid) {
    this->valid = valid;
}

std::string ValidationResponse::getMessage() {
    return message;
}

void ValidationResponse::setMessage(bool message) {
    this->message = message;
}
